modules/tinymce/src/core/main/ts/html/ParserFilters.ts

```diff
@@ -172,6 +173,24 @@ const register = (parser, settings: any): void => {
     });
   }
 
+  // Remove style attribute from styling elements and apply it to an inner span instead
+  parser.addNodeFilter('strong, b, em, i, u, strike', (nodes) => {
+    Arr.each(nodes, (node) => {
+      if (!node.attr('style')) {
+        return;
+      }
+      const span = Node.create('span', { style: node.attr('data-mce-style') || node.attr('style') });
```

Comment:
I don't know this section of the code very well, but I woulda thought we'd need to add both?

Reply:
Agreed, it should be a 1:1 copy.